docs/tutorials/get-started/cli.md

```diff
@@ -19,21 +19,21 @@ Local development is enabled through the `coda` command line tool (CLI). Keep re
 
 ## Before you get started
 
-To create a Pack you will need a Coda account, with [Doc Maker access][hc_doc_maker] in your workspace. If you're new to Coda, [sign up][sign_up] for a free account and you'll automatically be made a Doc Maker in your personal workspace.
-
 Make sure you have `node` and `npm` available on your machine. These two tools often come bundled together, and more information on how to install them is available in the [npm docs][npm_install].
 
-The instructions below assume some familiarity with the terminal / command prompt. If you aren't used to using this interface consult the help material for your operating system.
+--8<-- "tutorials/get-started/.cli/before.md"
 
 
-## Set up the Pack structure
+## Create a directory
 
-1. Create a directory for your new Pack.
+Installing the SDK and creating a Pack creates a lot of files, so you'll want to make sure you are working in a clean directory.
```

Comment:
Could potentially say "multiple files" instead of a "a lot of files". "A lot of files" sounds potentially more intimidating or annoying to me

Reply:
Done.